js/index.js

```diff
@@ -208,3 +208,30 @@ cardBtn.forEach((btn) => {
     openProjectModal(btn, closeProjectModal);
   });
 });
+
+/** *******************************************
+* Contact form validation
+********************************************* */
+const contactForm = document.querySelector('#contact-form');
+
+// Validate if email is lowercase
+const validateEmail = (email) => {
+  if (/[A-Z]/.test(email)) {
+    return false;
+  }
+  return true;
+};
+
+// Submit contact form with email validation
+contactForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const email = e.target.email.value;
+  const error = document.querySelector('.error');
+  const isSubmitOk = validateEmail(email);
+  if (isSubmitOk) {
+    error.style.display = 'none';
+  } else {
+    error.style.display = 'block';
+    error.innerHTML = 'Email must content only lower case charecters';
+  }
+});
```

Comment:
- [ ] The form email input validation works fine but when the lowercase email is entered and all input fields have values, the form is supposed to be submitted but it's not submitting at the moment. Kindly work on it 👍🏽

Reply:
Hi @AmaduKamara ,
Thanks for you nice review, you mean i need to submit the form using fetch API to formspree?